paima-demo/web-server/serve.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
```

Comment:
This script is taken from Godot tutorial - the same script that Godot editor uses when `Run in Browser` button is pressed.